### ISSUE TITLE
Allow user to specify CNVnator window size

### ISSUE THREAD

**Issue by s-boardman** [hall-lab/speedseq#38]
Particularly useful for low read sequencing analysis.

**Comment by s-boardman** [comment]
Proposed changes made on [s-boardman branch](https://github.com/s-boardman/speedseq/tree/s-boardman-cnvnator-window-size); currently testing before issuing pull request.

**Comment by cc2qe** [comment]
hey @s-boardman , try this out and see if it works for you

**Comment by s-boardman** [comment]
Hi @cc2qe, thanks for looking into this!

I've copied the speedseq & annotate_rd.py scripts to my local install of SpeedSeq and updated the config accordingly. Unfortunately I get the same error with 10000bp windows, but it still works fine with 6000bp ones?

I think those are the only two files I need to copy over? Did you have to recompile the source before it worked for you?

**Comment by cc2qe** [comment]
Yes, those are the only two files you should need to copy over. hang on, let me test further
